docs/scraping/providers/log-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+tags:
+  - Scraper
+  - Data
+---
+
+# Azure Log Analytics
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.9-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-No-red.svg)
+
+You can declare to scrape an Azure Log Analytics via the `LogAnalytics` resource type.
+
+The following fields need to be provided:
+
+- `name` - The name of the Azure Log Analytics
+- `workspaceId` - The workspaceId of the Log Analytics
+
+LogAnlaytics is different resource compare to others azure metric. So you need to replace azureMetricConfiguration field with logAnalyticsConfiguration field:
```

Comment:
```suggestion
The Azure Log Analytics scraper is configured differently compare to others Azure resources. Ensure that `logAnalyticsConfiguration` is configured instead of `azureMetricConfiguration`:
```